tarodb/sqlc/queries/assets.sql

```diff
@@ -237,6 +238,19 @@ JOIN genesis_info_view
 JOIN key_group_info_view
     ON assets.genesis_id = key_group_info_view.gen_asset_id;
 
+-- name: QueryGroups :many
+SELECT *
+FROM key_group_info_view
```

Comment:
So I think we can just query `asset_groups` directly here. There's only a single row in the groups for all the set of future instances. So we can just query with the tweaked key, then also the genesis info as well.

Reply:
Here's the query I had in mind when we were discussing this the other day:
```sql
SELECT genesis_assets.gen_asset_id, keys.raw_key, keys.key_index, keys.key_family 
FROM asset_groups groups
JOIN internal_keys keys ON groups.internal_key_id = keys.key_id
JOIN genesis_assets ON groups.genesis_point_id = genesis_assets.genesis_point_id
```

We're able to skip the intermediate join to `genesis_points`, since the group has the ref to the `genesis_point_id`, so we can just join with `genesis_assets` since they'll point to the same primary key. This should let us drop the order by and LIMIT below.

Reply:
Nvm, this runs into the same issue, current query should be fine.